RUFAS/biophysical/manure/single_stream_handler.py

```diff
@@ -0,0 +1,168 @@
+
+        Parameters
+        ----------
+        manure_stream : ManureStream
+            The manure to be processed.
+
+        Raises
+        ------
+        ValueError
+            If the ManureStream is incompatible with the processor receiving it.
+
+        """
+        info_map = {"class": self.__class__.__name__, "function": self.receive_manure.__name__}
+        super().receive_manure(manure_stream)
+        if self.manure_stream is not None:
+            self._om.add_error(
+                "Multiple stream received.",
+                f"Non parlor handler should only receive one manure stream at a time,"
+                f" handler {self.name} already received a manure stream.",
+                info_map,
+            )
+            raise ValueError("Handler cannot receive multiple streams.")
+        self.manure_stream = manure_stream
+
+    def process_manure(self, conditions: CurrentDayConditions, time: Time) -> dict[str, ManureStream]:
+        """
+        Executes the daily manure processing operations.
+
+        Parameters
+        ----------
+        conditions : CurrentDayConditions
+            Current weather and environmental conditions that manure is being processed in.
+        time : Time
+            Time instance containing the simulations temporal information.
+
+        Returns
+        -------
+        dict[str, ManureStream]
+            Mapping between classification of manure coming out of this processor to the ManureStream containing the
+            manure information. If the processor is a separator, the classifications are "solid" and "liquid". Otherwise
+            the only classification is "manure".
+
+        """
+        if self.manure_stream is None or self.manure_stream.pen_manure_data is None:
+            info_map = {"class": self.__class__.__name__, "function": self.process_manure.__name__}
+            self._om.add_error(
+                "None type ManureStream.",
+                "The processed ManureStream or pen data of the manure stream is None type.",
+                info_map,
+            )
+            raise TypeError("TypeError: Handler tries to process 'NoneType' object ManureStream.")
+        barn_temperature = self.determine_barn_temperature(conditions.mean_air_temperature)
+        surface_area = self.determine_barn_area(
+            self.manure_stream.pen_manure_data.animal_combination,
+            self.manure_stream.pen_manure_data.pen_type,
+            self.manure_stream.pen_manure_data.num_stalls,
+        )
+        self.ammonia_emission = self._calculate_ammonia_emissions(
+            self.manure_stream.ammoniacal_nitrogen,
+            self.manure_stream.volume,
+            990.0,
+            barn_temperature,
+            self.determine_ammonia_resistance(barn_temperature),
+            surface_area,
+            7.7,
+        )
+        return super().process_manure(conditions, time)
+
+    @classmethod
+    def determine_methane_emissions(
```

Comment:
```suggestion
    def determine_housing_methane_emissions(
```
Since we have other functions that calculate methane, it's helpful to specify that these are _housing_ methane emissions. I did not suggest this same change for ammonia because the same base ammonia calculation is used for both storage and housing ammonia emissions.